spec/controllers/v1/people_controller_spec.rb

```diff
@@ -2,26 +2,51 @@
 
 describe V1::PeopleController,  type: :controller do
   describe "POST create" do
-    it "returns success" do
-      user = instance_double("Person", id: 3, valid?: true)
-      allow(Person).to receive(:create_or_update).and_return(user)
-
-      post :create, person: { email: 'user@example.com', remote_fields: { tags: ['test'] } }
-      json_response = JSON.parse(response.body)
-
-      expect(Person).to have_received(:create_or_update)
-        .with(email: 'user@example.com', remote_fields: { tags: ['test'] })
-      expect(response).to be_success
-      expect(json_response).to have_key('id')
-      expect(json_response['id']).to eq(3)
+    render_views
+    context "with valid params" do
+      it "returns person" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.